### DISCUSSION TITLE
Touch screen support

### DISCUSSION THREAD

**Discussion by okias** [rwengine/openrw#766] [Ideas]
Running GTA 3 / Vice City data on computer is lame. PC's has lot of computing power and can emulate everything just fine.

What seems to be more challenging and valuable is running on low-power phones and tablets.

This won't be possible without basic touch screen support.

I'm putting it here for discussion and placeholder for me to implement it.

**Comment by tsjost** [comment]
The original purpose of openrw (#1) is to make GTA3D natively playable on platforms that it currently isn't.  
The GTA3D games were officially released for mobile on the 10 year anniversary, so it's not a target platform.

While it'd be easy to just accept touch events, we'd also have to make a separate touch-specific UI, and that doesn't seem very high up the priority list.
